src/nwb_datajoint/spikesorting/spikesorting_curation.py

```diff
@@ -1,61 +1,69 @@
 import json
 import os
 import shutil
+import time
 import uuid
 from pathlib import Path
+from typing import List
 from warnings import WarningMessage
 
 import datajoint as dj
+import numpy as np
 import spikeinterface as si
 import spikeinterface.toolkit as st
 
 from ..common.common_interval import IntervalList
 from ..common.common_nwbfile import AnalysisNwbfile
 from ..common.dj_helper_fn import fetch_nwb
+from .merged_sorting_extractor import MergedSortingExtractor
 from .spikesorting_recording import SpikeSortingRecording
 from .spikesorting_sorting import SpikeSorting
 
 schema = dj.schema('spikesorting_curation')
 
 
+def apply_merge_groups_to_sorting(sorting: si.BaseSorting, merge_groups: List[List[int]]):
+    # return a new sorting where the units are merged according to merge_groups
+    # merge_groups is a list of lists of unit_ids.
+    # for example: merge_groups = [[1, 2], [5, 8, 4]]]
+
+    return MergedSortingExtractor(parent_sorting=sorting, merge_groups=merge_groups)
+
+
 @schema
 class Curation(dj.Manual):
     definition = """
     # Stores each spike sorting; similar to IntervalList
-    curation_id: varchar(10)
+    curation_id: varchar(10) #A unique identifier for each curation entry for convenience
     -> SpikeSorting
     ---
+    curation_num = -1: int #
     parent_curation_id='': varchar(10)
     labels: blob # a dictionary of labels for the units
```

Comment:
is it a good idea to save these in the database? for large sortings this could be quite large

Reply:
I would give this a more descriptive name, e.g. `curation_labels`

Reply:
I don't think the size of the database is going to be a problem here. Each entry should still be < 1K, I would think and even 1000 of those is only 1MB.

Agreed on the names, though; we can change those.